tests/stubs/devices/mockserver.go

```diff
@@ -0,0 +1,52 @@
+package devices
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/kubeedge/kubeedge/tests/stubs/devices/services"
+	"github.com/paypal/gatt"
+	"github.com/paypal/gatt/examples/option"
+)
+
+func main() {
+	d, err := gatt.NewDevice(option.DefaultServerOptions...)
+	if err != nil {
+		log.Fatalf("Failed to open device, err: %s", err)
+	}
+
+	// Register optional handlers.
+	d.Handle(
+		gatt.CentralConnected(func(c gatt.Central) { fmt.Println("Connect: ", c.ID()) }),
+		gatt.CentralDisconnected(func(c gatt.Central) { fmt.Println("Disconnect: ", c.ID()) }),
+	)
+
+	// A mandatory handler for monitoring device state.
+	onStateChanged := func(d gatt.Device, s gatt.State) {
+		fmt.Printf("State: %s\n", s)
+		switch s {
+		case gatt.StatePoweredOn:
+			// Setup GAP and GATT services for Linux implementation.
+			// OS X doesn't export the access of these services.
+			d.AddService(services.NewGapService("sensortagmock")) // no effect on OS X
+			d.AddService(services.NewGattService())               // no effect on OS X
+
+			// Creating a temperature reading service
+			s1 := services.NewTemperatureService()
```

Comment:
s1 -> temperatureSvc

Reply:
updated

Reply:
i don't see the update ?

Reply:
Sorry...My bad..Missed it previously..updated..